packages/vue-client/src/components/GameView/DownloadSGF.vue

```diff
@@ -0,0 +1,26 @@
+
+  
+  <script setup lang="ts">
+  import { defineProps } from 'vue';
+  
+  const props = defineProps<{
```

Comment:
There is a TS error in CI:

```
error TS2345: Argument of type '{}' is not assignable to parameter of type 'NonNullable<(Partial<{ readonly [x: number]: string; }> & Omit<(readonly string[] | { readonly [x: string]: Prop<unknown, unknown> | null | undefined; }) & (VNodeProps & AllowedComponentProps & (ComponentCustomProps & Readonly<...>)), never> & Record<...>) | (Partial<...> & ... 1 more ... & Record<...>)> & Record<...>'.
```

It's a bit vague, but it will be resolved once you add some non-empty props here. (you could also fix the error by removing `defineProps()`, but I think we'll want to pass in `gameId` at least)

Reply:
Fixed!